spec/beaker/hypervisor/vsphere_spec.rb

```diff
@@ -58,6 +58,20 @@ module Beaker
 
       end
 
+      it 'does not provisions hosts if provision is set to false' do
+        MockVsphereHelper.powerOff
+        hosts = make_hosts()
+        hosts[0]["provision"] = false
+        vsphere = Beaker::Vsphere.new( hosts, make_opts )
+
+        vsphere.provision
+
+        hosts.each do |host|
+          expect( MockVsphereHelper.find_vm( host.name ).powerState ) == "poweredOff"
```

Comment:
it looks like you might be using the wrong syntax for the expect statement here.  When we're looking to test a matching state on an object, we typically use this syntax:

```ruby
expect( object.property ).to be === correct_answer
```

it looks like this line is missing the `.to be ===` section.  I'm not sure if what the `expect` method returns can be compared to a desired state.